WakeDevice/TargetEndpoint.cs

```diff
@@ -9,6 +9,9 @@
 
 namespace WakeDevice
 {
+    /// <summary>
+    /// Represents target endpoint network data
```

Comment:
*targets endpoint

Reply:
You may have misunderstood the concept behind that. An endpoint is a target itself, however there may be several endpoints and "target endpoint" simply specifies one particular endpoint. The word "target" is actually not necessary here. I'd stick to what it is now.